src/documentation/pages/components/tabs/examples/_horizontal.vue

```diff
@@ -0,0 +1,193 @@
+<template lang="pug">
+  <ComponentTabExample title="Horizontal" id="horizontal-lumen-centurylink" padding="-p--0" :tabs="exampleTabs" :menuTabs="menuTabs" :menuId="menuId" @toggleMenuId="toggleMenuId">
+    .-p--3.-bg--grey-20(:slot="$data.menuTabs[0].id")
```

Comment:
same here regarding paddings

Reply:
Padding is required here, if we remove -p--0 it creates an additional whitespace padding which is not rquired for this example.